save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/AgentService.kt

```diff
@@ -0,0 +1,57 @@
+package org.cqfn.save.orchestrator.service
+
+import org.cqfn.save.agent.HeartbeatResponse
+import org.cqfn.save.agent.NewJobResponse
+import org.cqfn.save.test.TestDto
+
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import reactor.core.publisher.Mono
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+
+/**
+ * Service for work with agents and backend
+ *
+ * @param baseUrl
+ */
+@Service
+// Fixme: delete suppress when resendTestsOnError function has any code
+@Suppress("EmptyFunctionBlock")
+class AgentService(baseUrl: String = "http://backend:5000") {
```

Comment:
Also, we decided to put all configuration parameters into property files and wrap them in configuration classes

Reply:
Added